examples/supertuxkart/main.go

```diff
@@ -0,0 +1,123 @@
+// Copyright 2020 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	sdk "agones.dev/agones/sdks/go"
+)
+
+type interceptor struct {
+	forward io.Writer
+}
+
+// Write will intercept the incoming stream, and forward
+// the contents to its `forward` Writer.
+func (i *interceptor) Write(p []byte) (n int, err error) {
+	return i.forward.Write(p)
+}
+
+// main intercepts the stdout and the log file of the Super Tux Kart gameserver and uses it
+// to determine if the game server is ready or not.
+func main() {
+	input := flag.String("i", "", "path to server_linux.sh")
+	flag.Parse()
+
+	fmt.Println(">>> Connecting to Agones with the SDK")
+	s, err := sdk.NewSDK()
+	if err != nil {
+		log.Fatalf(">>> Could not connect to sdk: %v", err)
+	}
+
+	fmt.Println(">>> Starting health checking")
+	go doHealth(s)
+
+	fmt.Println(">>> Starting wrapper for Super Tux Kart!")
+	fmt.Printf(">>> Path to Super Tux Kart server script: %s \n", *input)
+
+	cmdString := strings.Split(*input, " ")
+	command, args := cmdString[0], cmdString[1:]
+
+	cmd := exec.Command(command, args...) // #nosec
+	cmd.Stderr = &interceptor{forward: os.Stderr}
+	cmd.Stdout = &interceptor{forward: os.Stdout}
+
+	err = cmd.Start()
+	if err != nil {
+		log.Fatalf(">>> Error Starting Cmd %v", err)
+	}
+
+	// Super Tux Kart refuses to output to foreground, so we're going to
+	// poll the server log.
+	ready := false
+	home, err := os.UserHomeDir()
+	if err != nil {
+		log.Fatalf(">>> Could not get home dir %v", err)
+	}
+	for i := 0; i <= 10; i++ {
+		time.Sleep(time.Second)
+
+		logFile, err := os.Open(home + "/.config/supertuxkart/config-0.10/server_config.log")
```

Comment:
Maybe for a follow-on PR, but I wonder how static this location is and whether it should be settable via an env variable. 

In the mean time, Can you make this a constant instead of a magic string inline?